### PR TITLE
Add new cask `bili-downloader`

### PR DESCRIPTION
**Pull request by JounQin**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/bili-downloader.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  sha256 "ebfb5cb55f9662259b5fa2d9ffbcab93b5407b8b505c79c5bb3d47bfced85aed"
+
+  url "https://github.com/JimmyLiang-lzm/biliDownloader_GUI/releases/download/V#{version}/BiliDownloader_for_MacOS_X.dmg"
+  name "BiliDownloader"
+  desc "B站视频下载程序，BiliBili视频下载器，支持下载B站视频、交互\\互动视频，支持下载区域限制视频资源。"
+  homepage "https://github.com/JimmyLiang-lzm/biliDownloader_GUI"
+
+  livecheck do
+    url "https://github.com/JimmyLiang-lzm/biliDownloader_GUI/releases/latest"
+    strategy :github_latest
```
Comment (SMillerDev) [inline Casks/bili-downloader.rb:12]: This doesn't need the `:github_latest` strategy then.

Comment (SMillerDev) [reply Casks/bili-downloader.rb:12]: Yes indeed

Comment (JounQin) [reply Casks/bili-downloader.rb]: @SMillerDev See https://github.com/Homebrew/homebrew-cask/runs/6595226283?check_suite_focus=true#step:11:12

**Comment by miccal** [comment]
Thank you @JounQin.

**Review by miccal — APPROVED** [review]

**Pull request merged**